[docs] Faucet update (#20958)
MystenLabs/sui: docs/content/guides/developer/getting-started/get-coins.mdx

title: Get SUI Tokens
---

Sui faucet is a helpful tool where Sui developers can get free test SUI tokens to deploy and interact with their programs on Sui's Devnet and Testnet networks. There is no faucet for Sui Mainnet...

## Prerequisites

To request tokens from the faucet, you must own a wallet address that can receive the SUI tokens. See the [Get Sui Address](./get-address.mdx) topic if you don't already have an address or need to create a new one.

## Request test tokens through the online faucet {#online-faucet}

An online faucet is available for test token requests.

**URL:** https://faucet.sui.io/

> Connect your wallet or paste your wallet address in the address field. Use the network dropdown to select the correct network and click the Request SUI button.
> 
> You can refresh your browser and click the Request SUI button again to quickly repeat the process. The requests are rate limited, however, so too many requests results in a waiting period before you are able to request more tokens. 

Reviewer: ```suggestion
** How to use it **
- connect your wallet or paste your wallet address in the address field
- use the network dropdown to select the correct network 
- click the Request SUI button. 

To request more SUI, refresh your browser and click the Request SUI button again. The requests are rate limited, however, so too many requests results in a waiting period before you are able to request more tokens. 
```

### Return unused Testnet SUI

The Testnet faucet drains from a finite pool of SUI. If the pool empties, it disrupts faucet service for the rest of the community. To help ensure this doesn't happen, you can use the online faucet to return your unused SUI to the pool.

> If you connect your wallet that contains the unused SUI to the online faucet, you can click the **Return tokens to faucet** button. Approve the transaction using your wallet and your Testnet SUI are returned to the pool. 
> 
> If you prefer not to connect your wallet, you can click the **Copy** button to the right of the return address and send the tokens via a separate transaction.
> 
> When you need those tokens again, just revisit the faucet.

Reviewer: ```suggestion
There are two ways to return unused Testnet SUI:
- connect your wallet in the online faucet, and click the **Return tokens to faucet** button. Approve the transaction using your wallet and your Testnet SUI are returned to the pool.
- if you prefer not to connect your wallet, you can click the **Copy** button to the right of the return address and send the tokens via a separate transaction.
```
